untie/src/untie.cpp

```diff
@@ -173,6 +180,9 @@ Untie::Untie() {
         vec3 halfsize = vec3(1.5f);
         shape.set_geometry(shape.create_cube_geometry(halfsize));
 
+        auto& aggregate = cubeObj.add_component<components::Aggregate>();
```

Comment:
Looking at the documentation are aggregates not about being able to maintain a collection of rigidbodies i.e. a ragdoll, as opposed to a one to one basis? if that's not the case, what are their use?

Reply:
it is for the collision filter. A good way for particles system and level design.

Reply:
Okay yeah that makes sense, I think we're going to scrap physical simulation of particles though we don't have the time to have it done. But understand what you mean for level design as the aggregate turns of self collisions. The issue there is though if we have one aggregate for the whole level then in the narrow phase the entire level must be checked against a player on it. We could use chunking to solve this but maybe a bit annoying to do manually?

Should this really also be tied into the hierarchy system? @lmariscal  or this might be too much work for how long we have left?

Reply:
I think the player and the leve such as wall and floor should be in different aggregates. The aggregate makes sure the wall and floor do not move away because the  collision when we build the scene

Reply:
It would be really nice to have it tied with the Hierarchy system, but not sure about the time it would take.